lib/l10n/intl_ro.arb

```diff
@@ -31,13 +31,15 @@
   "labelEvaluation": "Evaluare",
   "labelPoints": "Puncte",
   "labelPrivacyPolicy": "Privacy Policy",
+  "labelPersonalInformation": "Informații personale",
 
   "sectionShortcuts": "Scurtături",
   "sectionEvents": "Evenimente",
   "sectionFrequentlyAccessedWebsites": "Website-uri favorite",
   "sectionEventsComingUp": "Evenimente următoare",
   "sectionFAQ": "Întrebări frecvente",
   "sectionGrading": "Punctaj",
+  "sectionEditProfile": "Setările contului",
```

Comment:
Either you call this "Modifică profil" so it corresponds to the English version, or you change the English version and the variable name to "Account settings".